src/execution/command_path.c

```diff
@@ -0,0 +1,50 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   command_path.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ramoussa <ramoussa@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/26 19:31:03 by ramoussa          #+#    #+#             */
+/*   Updated: 2023/12/26 19:46:12 by ramoussa         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <unistd.h>
+#include "minishell/execution/executor.h"
+
+static char	*find_path_in_env(char **envp, char *program)
+{
+	int		idx;
+	char	*path;
+	char	**env_paths;
```

Comment:
Can you align these variables so they follow norminette?

Reply:
yeah right i will handle norminette